accesslog/middleware.go

```diff
@@ -56,8 +59,16 @@ func (mw *AccessLogMiddleware) MiddlewareFunc(h rest.HandlerFunc) rest.HandlerFu
 
 		logger := requestlog.GetRequestLogger(r)
 		util := &accessLogUtil{w, r}
-
-		logger.Print(mw.executeTextTemplate(util))
+		logger.WithFields(logrus.Fields{
+			"type":         TypeHTTP,
+			"ts":           util.StartTime().Round(0),
+			"status":       util.StatusCode(),
+			"responsetime": util.ResponseTime().Seconds(),
+			"bytes":        util.BytesWritten(),
```

Comment:
nitpick2: I would change to `byteswritten` from "bytes"

Reply:
done